### PR TITLE
document processing trigger access

### PR DESCRIPTION
**Pull request by Ortega-Dan**
Bug Ticket: https://dol-ewf.atlassian.net/browse/DSG-1594

## Description

Document processing trigger access for public user on its own transactions

## Motivation and Context

Discussion https://nuvalence.slack.com/archives/C04HWDTRCMT/p1692896841545479

## How Has This Been Tested?

Integrated and API tests

## Types of changes

Patch
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation change (documentation only, will not require redeployment)
- [ ] Refactor or cleanup (functionality unchanged)
Minor
- [ ] New feature (non-breaking change which adds functionality)
Major
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have performed a self-review of my own code
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

- [x] I have confirmed all acceptance criteria defined by the story have been met

### CODE REVIEW

File: helm/work-manager/cerbos-policies/transaction_resource.yaml [hunk 1]
```diff
@@ -12,6 +12,13 @@ resourcePolicy:
       effect: EFFECT_ALLOW
 
     - actions: ["view", "update"]
+      roles:
+        - transaction-admin
```
Comment (davidyoung77) [inline helm/work-manager/cerbos-policies/transaction_resource.yaml:16]: To verify does this allow the public user to post to the endpoint?

**Review by davidyoung77 — APPROVED** [review]
LGTM, i feel comfortable approving this, pending comment

File: helm/work-manager/cerbos-policies/transaction_resource.yaml (hunk 1 above)
Comment (JayStGelais) [reply helm/work-manager/cerbos-policies/transaction_resource.yaml:16]: Yes, for their transactions only.

**Review by asultan001 — APPROVED** [review]
LGTM

**Pull request merged**